### PR TITLE
Feature/data explorer change wording

### PR DESCRIPTION
**Pull request by weberjavi**
Changed `Emission Pathways` labels to `Pathways` on Data Explorer component.
Also reworded `Regions` dropdown label to `Countries and regions` 
[Pivotal (pathways)](https://www.pivotaltracker.com/story/show/159405777)
[Pivotal (Countries and regions)](https://www.pivotaltracker.com/story/show/159404553)
test here:
http://localhost:3000/data-explorer/historical-emissions?page=1

image
image

### CODE REVIEW

**Comment by weberjavi** [comment]
Would be nice to get rid of `emission-pathways` in all the project (both on code and on files/folder structure)

File: app/javascript/app/components/data-explorer-content/data-explorer-content-component.jsx [hunk 1]
```diff
@@ -84,8 +84,8 @@ class DataExplorerContent extends PureComponent {
         (multipleSector(field) ? (
           <MultiDropdown
             key={field}
-            label={toStartCase(field)}
-            placeholder={`Filter by ${toStartCase(field)}`}
+            label={deburrCapitalize(field)}
```
Comment (Bluesmile82) [inline app/javascript/app/components/data-explorer-content/data-explorer-content-component.jsx]: Please use the toStartCase util method. This is basically Converting some acronyms like INDC or CO2 to the correct format

File: app/javascript/app/data/constants.js [hunk 2]
```diff
@@ -310,6 +310,13 @@ export const DATA_EXPLORER_SECTION_NAMES = {
   'ndc-content': 'ndc_content'
 };
 
+export const DATA_EXPLORER_SECTION_LABELS = {
```
Comment (Bluesmile82) [inline app/javascript/app/data/constants.js]: Maybe we can rename DATA_EXPLORER_SECTION_NAMES to URIS or maybe even merge them as they have the same keys. Having Labels and Names together could be confusing

**Comment by Bluesmile82** [comment]
Renaming the column has affected the flow of the dropdown data. Please take a look:

image

**Comment by weberjavi** [comment]
Ok so I've misunderstood the requirement for this task ([change Regions to Countries and regions](https://www.pivotaltracker.com/story/show/159404553)), thought it was just a wording thing:

image

What they probably want is to display also countries on the dropdown options, as we are doing on GHG emissions module. Also to be consistent a rephrasing of the dropdown label of the module should be done.

image

I'll revert those changes and create another task to address them in other PR.
What do you think @Bluesmile82, @simaob ?

File: app/javascript/app/data/constants.js (hunk 2 above)
Comment (weberjavi) [reply app/javascript/app/data/constants.js]: If you agree I'll go with the change of the constant to `DATA_EXPLORER_SECTION_URIS`.

**Comment by Bluesmile82** [comment]
Yes, I think you are right. We will have to mimic the dropdown. It's a bigger issue that should be addressed in other PR

File: app/javascript/app/components/data-explorer-content/data-explorer-content-component.jsx (hunk 1 above)
Comment (weberjavi) [reply app/javascript/app/components/data-explorer-content/data-explorer-content-component.jsx]: Do we need to make those substitutions on this dropdown labels? 
Also what we need here is capitalize the string, using `toStartCase` returns `Countries And Regions` while `deburrCapitalize` returns `Countries and regions`

Even do acronym replacements inside a function called `toStartCase` may be hiding the real functionality of the function, shouldn't it be better to compose two functions and do something like `startCase(replaceAll(string, ACRONYM_REPLACEMENTS))`.

File: app/javascript/app/data/constants.js (hunk 2 above)
Comment (weberjavi) [reply app/javascript/app/data/constants.js]: ⚡️ done

**Pull request merged**